Makefile

```diff
@@ -32,9 +34,21 @@ sizing-styles:
 	@echo "Generating sizing style definitions..."
 	@node genSizing.js
 
+LINT_MAX_LESS_PROBLEMS := 127
 lint:
 	@echo "Linting files..."
 	@$(LINT) $(JS_FILES) $(JSX_FILES)
+	@$(STYLELINT) $(LESS_FILES) | tee ./tmp/less-lint-output.txt || true
```

Comment:
we should output these to `/tmp/...` instead of `./tmp/...` so it doesn't end up in the project folder